tests-execution-chart-commons/src/test/groovy/io/github/platan/tests_execution_chart/reporters/json/JsonReporterTest.groovy

```diff
@@ -0,0 +1,53 @@
+package io.github.platan.tests_execution_chart.reporters.json
+
+import io.github.platan.tests_execution_chart.report.data.Mark
+import io.github.platan.tests_execution_chart.report.data.TestExecutionScheduleReport
+import io.github.platan.tests_execution_chart.report.data.TimedTestResult
+import spock.lang.Specification
+import java.time.Instant
+
+import static io.github.platan.tests_execution_chart.report.data.TimedTestResult.Type.TEST
+
+class JsonReporterTest extends Specification {
+    def "should generate report in json"() {
+        given:
+        def PATHNAME = "./test"
```

Comment:
Can we make `PATHNAME` and `CONFIG_OUTPUT_LOCATION` lowercase?

Reply:
yes